app/scripts/lib/multichain-api/scope/validation.ts

```diff
@@ -36,21 +35,13 @@ export const isValidScope = (
 
   // These assume that the namespace has a notion of chainIds
   if (reference && scopes && scopes.length > 0) {
-    // TODO: Probably requires refactoring this helper a bit
-    // When a badly-formed request includes a chainId mismatched to scope
-    //   code = 5203
-    //   message = "Scope/chain mismatch"
-    // When a badly-formed request defines one chainId two ways
-    //  code = 5204
-    //  message = "ChainId defined in two different scopes"
```

Comment:
I don't think we need to throw these. Our current strategy has been to drop/ignore malformed ScopeObjects, and so these sort of errors don't really apply to us

Reply:
I think we have landed on not firing these